### DISCUSSION TITLE
Can you 'chain' tools / pipelines ?

### DISCUSSION THREAD

**Discussion by jabbasj** [open-webui/open-webui#3285] [Q&A]
Does the feature for tools to trigger other tools automagically exist already ? 

Like User > LLM >  Tool A > LLM > Tool B > LLM > Tool C > LLM > User

**Comment by jabbasj** [comment]
Please does anyone know ?

**Reply by tjbck** [reply]
Uncharted territory but you probably could, please keep us updated on your journey 🫡

**Reply by jjohare** [reply]
I need this at some point. The way I was going to do it was through the external FastAPI called to OpenWebUI. I think that's the easiest way for you @jabbasj... basically just script it.
